loginsystem/newsfeed/images/News Feed.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<!-- saved from url=(0069)file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/splashpage.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="stylesheet" type="text/css" href="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/CSS/style.css?version=1">
+
+  <title>News Feed</title>
+
+</head>
+<body>
+
+<div class="grid-container">
+  <div class="item1"><img src="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/images/newcolorlogo.png" width="200" height="170" alt=""></div>
```

Comment:
to avoid having comments on every line, I'll just put a general comment here for the names created. please go back and look through them to make sure that they  are relatable to the context of the elements being created like we talked about yesterday.